tests/testthat/test_bundles.R

```diff
@@ -1,30 +1,14 @@
-test_that("'.index_GET()' works for bundles", {
-
-    bundles_titles <- c("Tabula Muris: Transcriptomic characterization of
-                     20 organs and tissues from Mus musculus at single cell
-                     resolution", "A Single-Cell Transcriptomic Map of the
-                     Human and Mouse Pancreas Reveals Inter- and Intra-cell
-                     Population Structure")
-    bundles_filters <- filters(projectTitle = list(is = bundles_titles))
-
-    bundles_resp <- .index_GET(filters = bundles_filters,
-                               size = 100,
-                               sort = "projectTitle",
-                               order = "asc",
-                               catalog = "dcp2",
-                               base_path = "/index/bundles")
-    expect_equal(bundles_resp$status_code, 200)
+test_that("'bundles()' works with default arguments", {
+    test_proj <- bundles()
```

Comment:
not quite in the spirit of the title of the test, but using a small e.g., `size = 3` would likely make the tests faster and more robust without compromising value?